### PR TITLE
Fix and remove two outdated links in JS How this course will work lesson

### PR DESCRIPTION
**Pull request by pantbruin**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [ ] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:
The first "Foundations course" link includes a hash mark in the URL, linking to a section that no longer exists. While the link is not necessarily broken, the hash link to the specific subsection should be removed for consistency with the curriculum. Further, second link "Front End Section" was removed as this section is nonexistent in the Foundations curriculum. 

#### 2. Related Issue
N/A

### CODE REVIEW

**Review by fortypercenttitanium — CHANGES_REQUESTED** [review]
Can you just change the links to the Javascript basics section (https://www.theodinproject.com/paths/foundations/courses/foundations#javascript-basics) and maybe reword a bit so that the link makes sense?

**Comment by pantbruin** [comment]
Fixed!

**Review by fortypercenttitanium — APPROVED** [review]
Awesome, thank you!
